UI/sign-in/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
```

Comment:
It's very cool to have unit test! Could more tests be added? e.g.
it ('should show SignIn as the default view')
it ('should change view when calling handleClick()')